### DISCUSSION TITLE
Issues-only permissions

### DISCUSSION THREAD

**Discussion by max-prtsr** [community/community#4298] [Projects and Issues]
`Reader` role has access to the code, which we want to hide from our QA team. 
Is it possible to fix it somehow? Now we have a second repo for QA stuff and it's very inconvenient to track both repos issues.

**Comment by DavidMarquezF** [comment]
I'm surprised not more people have upvoted this, it is so convenient to have issues directly in the repo but having to give access to the code to the Support Team seems like a very bad idea

**Comment by matthewcobb** [comment]
We would like to have a single project management solution. Currently we are stuck between basecamp (for stakeholders and support team) and Github projects for developers. Having Issue only permission access would help a lot; as mentioned we don't want to leak source code.
